docs/source/markdown/podman-create.1.md

```diff
@@ -613,6 +615,8 @@ Current supported mount TYPEs are **bind**, **volume**, **image**, **tmpfs** and
 
 	      · ro, readonly: true or false (default).
 
+        . U: true (default).
```

Comment:
It feels weird to only have shorthand option for this. I think it should be called `chown` as well. Also it is missing a description what this option does.
You also say the default is true but when the option is not set the ownership is not changed, so the default is false.

Reply:
done